blog/assets/webinars/Dec2024/Dec11-Webinar-AKS_Feature_Announcements_from_KubeCon_Ignite .ics

```diff
@@ -0,0 +1,303 @@
+BEGIN:VCALENDAR
+PRODID:-//Microsoft Corporation//Outlook 16.0 MIMEDIR//EN
+VERSION:2.0
+METHOD:REQUEST
+X-MS-OLK-FORCEINSPECTOROPEN:TRUE
+BEGIN:VTIMEZONE
+TZID:Pacific Standard Time
+BEGIN:STANDARD
+DTSTART:16011104T020000
+RRULE:FREQ=YEARLY;BYDAY=1SU;BYMONTH=11
+TZOFFSETFROM:-0700
+TZOFFSETTO:-0800
+END:STANDARD
+BEGIN:DAYLIGHT
+DTSTART:16010311T020000
+RRULE:FREQ=YEARLY;BYDAY=2SU;BYMONTH=3
+TZOFFSETFROM:-0800
+TZOFFSETTO:-0700
+END:DAYLIGHT
+END:VTIMEZONE
+BEGIN:VEVENT
+ATTENDEE;CN="Webinar AKS Feature Announ";RSVP=TRUE:invalid:nomail
+ATTENDEE;CN="Sanket Bakshi";RSVP=TRUE:mailto:Sanket.Bakshi@microsoft.com
+ATTENDEE;CN="Cory Skimming";RSVP=TRUE:mailto:coryskimming@microsoft.com
+ATTENDEE;CN="Brian Redmond";RSVP=TRUE:mailto:Brian.Redmond@microsoft.com
+ATTENDEE;CN="Jorge Palma";RSVP=TRUE:mailto:Jorge.Palma@microsoft.com
+CLASS:PUBLIC
+CREATED:20241203T181139Z
+DESCRIPTION:Organizers & co-organizers: When making edits to your event\, p
+	lease use Teams. <https://nam06.safelinks.protection.outlook.com/?url=http
+	s%3A%2F%2Fteams.microsoft.com%2Fl%2Fvirtualevent%2F098901a7-31df-4c1f-aac7
+	-5d2d15bae5d8%4072f988bf-86f1-41af-91ab-2d7cd011db47&data=05%7C02%7Ccorysk
+	imming%40microsoft.com%7C9e890e3880cb4392c52d08dd12f4f109%7C72f988bf86f141
+	af91ab2d7cd011db47%7C1%7C0%7C638687565401699845%7CUnknown%7CTWFpbGZsb3d8ey
+	JFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIs
+	IldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=GltF7XuC9d8CTtD7pLz17QBEXbIYXl8LshAv68
+	KAPzI%3D&reserved=0>  	\n\nWebinar: AKS Feature Announcements from KubeCon
+	 & Ignite \n\n\nYou’ve been invited to: \n\n\nWebinar: AKS Feature Annou
+	ncements from KubeCon & Ignite \n\nJoin us for a 1 hr webinar event where 
+	we discuss all the recent announcements for the Azure Kubernetes Service (
+	AKS) that were done at the KubeCon North American and Ignite 2024 conferen
+	ces. \n\n \n\nWed\, Dec 11\, 2024 8:00 AM - 9:00 AM (UTC-08:00) Pacific Ti
+	me (US & Canada)\n\n\nWe look forward to having you attend the event!\n\n\
+	n_________________________________________________________________________
+	_______\nMicrosoft Teams Need help? <https://nam06.safelinks.protection.ou
+	tlook.com/?url=https%3A%2F%2Faka.ms%2FJoinTeamsMeeting%3Fomkt%3Den-US&data

```

Comment:
This can be shortened no? It's the outlook link protection stuff?

Reply:
Removed safelinks from ics file, tested and re-committed.